samcli/commands/deploy/command.py

```diff
@@ -30,18 +36,21 @@
 \b
 """
 
+CONFIG_SECTION = "parameters"
+
 
 @click.command(
     "deploy",
     short_help=SHORT_HELP,
     context_settings={"ignore_unknown_options": False, "allow_interspersed_args": True, "allow_extra_args": True},
     help=HELP_TEXT,
 )
-@configuration_option(provider=TomlProvider(section="parameters"))
+@configuration_option(provider=TomlProvider(section=CONFIG_SECTION))
```

Comment:
I get what you're doing here, but I wonder if this should be a shared constant, and you're saying `CONFIG_PARAM_SECTION`, because if there are ever other sections, this won't be easy to understand.

Your call on this, it can always be refactored later.

Reply:
I called out on this, as well. later refactor